test/Activity-test.js

```diff
@@ -6,7 +6,16 @@ import User from '../src/User';
 describe('Activity', function() {
   let activity;
   let user;
+  let user1;
+  let user2;
+  let activity1;
+  let activity2;
   let userRepository;
+  let userData;
+  let sleepData;
+  let activityData;
+  let hydrationData;
+
```

Comment:
Do we need these if we can just declare them in their own lexical scope?

Reply:
We need them to instantiate the UserRepository.